mobius-core/src/main/java/com/spotify/mobius/MobiusLoop.java

```diff
@@ -140,7 +140,7 @@ public void accept(E event) {
   public void dispatchEvent(E event) {
     if (disposed)
       throw new IllegalStateException(
-          "This loop has already been disposed. You cannot dispatch events after disposal");
+          "This loop has already been disposed. You cannot dispatch events after disposal - event received: " + event + ", currentModel: " + mostRecentModel);
```

Comment:
not really important but maybe String.format would be nice here?

Reply:
Updated, plus added the event type as well, to be less dependent on a `toString` with sufficient information.